checkers-backend/src/api/game/game.controller.spec.ts

```diff
@@ -0,0 +1,18 @@
+import { Test, TestingModule } from '@nestjs/testing';
```

Comment:
nice idea to add unit test's to controllers, but currently we are not testing here anything. In future will be nice to test in those tests some logic in module. But you do not need to do  this now :)